### ISSUE TITLE
[1.20.4] - Immersive Engineering Compatability Issues

### ISSUE THREAD

**Issue by AriaElidove** [emilyploszaj/emi#605]
There are a few issues regarding Immersive Engineering and Just Enough Items as well as EMI that simply do not happen without EMI. There may be more, there may be less, I would not know anything beyond what I have found and am listing below.

----
1) Invalid Enchantments

EMI registers that certain non-enchantable items can be enchanted. A couple examples of this are the Collapsible Glider and Wirecutters in Immersive Engineering. EMI assumes that the Collapsible Glider can be enchanted with generic enchantments, such as Mending, Unbreaking, and Curse of Vanishing & Binding, but this is not the case when one would attempt to do so via anvil. Just Enough Items, when alone with Immersive Engineering, will show that the enchantment is not true.

----
With EMI
![image](https://github.com/emilyploszaj/emi/assets/144502047/64c6db28-d67b-4739-b165-d072518eff71)

----
Without EMI (Note the lack of an Anvil)
![image](https://github.com/emilyploszaj/emi/assets/144502047/47cf0dfe-ecfe-49e1-a85f-9f727469d77d)

----
2) Custom Recipes

Immersive Engineering has many different custom recipes, but none are registered with EMI itself. These are instead registered with Just Enough Items. When EMI is included with these two mods, it causes Immersive Engineering's recipes to break. Certain information and values are left completely unmarked or return as outright incorrect data. In short, there are instances where the output value simply does not exist, and only the item in context appears.

----
With EMI
![image](https://github.com/emilyploszaj/emi/assets/144502047/1e8aac38-7b56-4851-9a28-c1836168d837)

----
Without EMI (Note the output counts)
![image](https://github.com/emilyploszaj/emi/assets/144502047/0549370b-e5ec-44b6-b67a-0be84cb90bc8)

----
TLDR; EMI makes assumptions about certain items and the enchantments that they can hold, and EMI fails to provide correct output values for custom recipes.

This has been reported with Immersive Engineering already: https://github.com/BluSunrize/ImmersiveEngineering/issues/5968

**Comment by AriaElidove** [comment]
The second half is a duplicate of #597 for 1.20.4
